manifests/0000_05_config-operator_01_authentication.crd.yaml

```diff
@@ -13,6 +14,12 @@ spec:
   subresources:
     status: {}
   versions:
-    - name: v1
-      served: true
-      storage: true
+  - name: v1
+    served: true
+    storage: true
+status:
```

Comment:
I'm not clear on why we're doing this, but ok.

Reply:
fixed. That were default values that yaml marshalling added. Removed them by not going via CRD golang types.